repos/nuget-client.proj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <EnvironmentVariables Include="MS_PFX_PATH=$(NuGetKeyFilePath)" />
+    <EnvironmentVariables Include="NUGET_PFX_PATH=$(NuGetKeyFilePath)" />
```

Comment:
I wasn't sure about this when I first saw it. I see now it comes from https://github.com/dotnet/source-build/pull/598, so we probably do need it. @crummel

Reply:
Yeah, there's now two different variables used for the PFX path, we will attempt to delay-sign and fail without this.  I can look at whether we can disable signing entirely, that seems like the better option.

Reply:
Cool, filed https://github.com/dotnet/source-build/issues/647.